frontend/src/pages/NewStudent.vue

```diff
@@ -175,17 +177,14 @@ export default {
         StatusString: "SCREENING",
       })
         .then((response) => response.json())
-        .then((data) => {
-          this.$buefy.snackbar.open({
-            message: "New Student Added",
-            type: "is-success",
-            position: "is-top",
-            actionText: "View Profile",
-            indefinite: true,
-            onAction: () => {
-              this.$router.push({ path: `/students/${data.StudentID}` });
-            },
-          });
```

Comment:
As discussed, this should not have been removed.

Reply:
Restored.